### PR TITLE
Endpoint connectivity error display adjustment

### PR DESCRIPTION
**Pull request by sergioc**

### CODE REVIEW

File: src/ServicePulse.Host/app/modules/monitoring/js/endpoint_details.controller.js [hunk 1]
```diff
@@ -72,7 +74,9 @@
                         }
                     }, function (err) {
                         // Warn user?
-                    });
+                        });
+
+                    $scope.endpoint.isStale = $scope.endpoint.isStale && instance.isStale;
```
Comment (WilliamBZA) [inline src/ServicePulse.Host/app/modules/monitoring/js/endpoint_details.controller.js]: Should this not be an `||`? Is an endpoint stale only if all the instances are stale or is it stale if at least one instance is stale?

Comment (tmasternak) [reply src/ServicePulse.Host/app/modules/monitoring/js/endpoint_details.controller.js]: The endpoint is stale if all instances are stale => `isStale` is true for all instances => `&&`.

Comment (WilliamBZA) [reply src/ServicePulse.Host/app/modules/monitoring/js/endpoint_details.controller.js]: :+1:

**Comment by WilliamBZA** [comment]
Has conflicts. I also think a milestone and a label (feature or bug) is needed for some of these PRs so that the release notes builder knows what to do

**Comment by tmasternak** [comment]
@WilliamBZA rebased.

**Pull request merged**